vllm/model_executor/layers/linear.py

```diff
@@ -471,6 +493,81 @@ def weight_loader(self,
         assert param_data.shape == loaded_weight.shape
         param_data.copy_(loaded_weight)
 
+    def _default_loading(self, param: vLLMParameter, param_data: torch.Tensor,
+                         loaded_weight: torch.Tensor, loaded_shard_id: int):
```

Comment:
Instead of `_default_loading`, we could call this `load_weight_tensor`, since this function holds the logic for loading the weight (as opposed to the scale or metadata in quantized case)